chromadb/types.py

```diff
@@ -219,6 +219,25 @@ class LogRecord(TypedDict):
     record: OperationRecord
 
 
+class RequestVersionContext(TypedDict):
+    """The version and log position of the collection at the time of the request
+
+    This is used to ensure that the request is processed against the correct version of the collection.
+
+    For example, if the FE first queries the metadata segment and then queries the vector segment, the version
```

Comment:
nit: maybe also add a line on why both the version and log position are needed?

Reply:
sg